connect-examples/v2/php_checkout/.env

```diff
@@ -1,15 +1,9 @@
-# Acceptable values are sandbox, production or custom. 
-ENVIRONMENT=sandbox
-
-# Production credentials
-PRODUCTION_ACCESS_TOKEN=production-access-token
-PRODUCTION_APP_ID=production-app-id
-PRODUCTION_LOCATION_ID=production-location-id
-
-# Sandbox credentials
-SANDBOX_ACCESS_TOKEN=sandbox-access-token
-SANDBOX_APP_ID=sandbox-app-id
-SANDBOX_LOCATION_ID=sandbox-location-id
+# Acceptable values are sandbox or production
+ENVIRONMENT=production|sandbox
```

Comment:
I think it might not be clear that the current value is a placeholder. What do you think about leaving it blank and mentioning in the comment that this needs to be filled out?

Reply:
Same for the `ENVIRONMENT` variable in other `.env` files